client/src/commons/PhoneNumberTextField/PhoneNumberTextField.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {TextField, StandardTextFieldProps} from '@material-ui/core';
+
+import CircleTextField from 'commons/CircleTextField/CircleTextField';
+import Validator from 'Utils/Validations/Validator';
+
+const PHONE_LABEL = 'טלפון:';
+const NOT_PHONE_ERROR = 'שגיאה: מספר שהוזן אינו תקין';
+
+const PhoneNumberTextField: React.FC<Props> = (props: Props): JSX.Element => {
+    const [isError, setIsError] = React.useState<boolean>(false);
+
+    const {id, value, onChange, testId, ...rest } = props;
+
+    return (
+        <CircleTextField
+            {...rest}
+            error={isError}
+            onBlur={() => setIsError(value && !Validator.phoneValidation(value as string))}
+            onFocus={() => setIsError(false)}
+            label={isError && NOT_PHONE_ERROR}
+            test-id={testId}
+            id={id}
+            placeholder={PHONE_LABEL}
+            size='small'
+            value={value}
+            onChange={onChange}
+        />
+    );
+};
+
+export default PhoneNumberTextField;
+
+interface Props extends StandardTextFieldProps {
+    onChange: (event: any) => void;
+    testId?: string;
```

Comment:
I think OnChange is already on the textFieldProos, in addition you should extend the CircleTextFIeldProps instead